backend/src/types.ts

```diff
@@ -265,7 +265,7 @@ export type KubeFastifyInstance = FastifyInstance & {
 
 // TODO: constant-ize the x-forwarded header
 export type OauthFastifyRequest<Data extends RouteGenericInterface = RouteGenericInterface> =
-  FastifyRequest<{ Headers: { 'x-forwarded-access-token': string } & Data['Headers'] } & Data>;
+  FastifyRequest<{ Header?: { 'x-forwarded-access-token'?: string } & Data['Headers'] } & Data>;
```

Comment:
Uh -- isn't it `Headers`? 😕 

That's fair that it's optional -- local development...